src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/SpecificationsType.php

```diff
@@ -44,6 +67,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('references', ReferencesType::class)
             ->add('features', FeaturesType::class)
             ->add('attachments', ProductAttachmentsType::class)
+            ->add('show_condition', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Display condition on product page', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h3',
+                'show_choices' => false,
+                'inline_switch' => true,
+            ])
+            ->add('condition', ChoiceType::class, [
+                'choices' => $this->productConditionChoiceProvider->getChoices(),
+                'attr' => [
+                    'class' => 'custom-select',
+                ],
+                'required' => false,
+                // placeholder false is important to avoid empty option in select input despite required being false
+                'placeholder' => false,
+                'label' => false,
```

Comment:
is there a reason to set the label to false ? (it had a 'condition' string when it was in  `OptionsType` )

Reply:
Yes in the new design this input no longer has a label, we only have a label on the checkbox before

Reply:
![Capture d’écran 2022-01-10 à 10 53 32](https://user-images.githubusercontent.com/13801017/148746784-7936dcf4-2b01-46ab-b192-e563186bf1d6.png)

Reply:
@matthieu-rolland is it legit with this explanation? ^^